openedx/features/learner_profile/static/learner_profile/fixtures/learner_profile.html

```diff
@@ -1,19 +1,38 @@
 <div class="message-banner" aria-live="polite"></div>
 <div class="wrapper-profile">
-  <div class="ui-loading-indicator">
-    <p>
+    <div class="profile profile-other">
+        <div class="wrapper-profile-field-account-privacy"></div>
+        <div class="wrapper-profile-sections account-settings-container">
+            <div class="wrapper-profile-section-container-one">
+                <div class="wrapper-profile-section-one">
+                    <div class="profile-image-field">
+                    </div>
+                    <div class="profile-section-one-fields">
+                    </div>
+                </div>
+                <div class="ui-loading-error is-hidden">
+                    <span class="fa fa-exclamation-triangle message-error" aria-hidden="true"></span>
+                    <span class="copy">An error occurred. Try loading the page again.</span>
```

Comment:
That makes sense, and I thought there was one, but I can't find it. Part of the challenge is that we render the indicator server side so that it appears immediately, but we remove it client-side when the page is complete. This is the kind of thing that server-side React would be awesome for...

Reply:
Yes. Sorry that wasn't clear. I think it isn't possible to do with the current mix of server-side and client-side code. We could have helpers on each side but I don't think that would buy much. I suppose we could have a Mako template for the loading error which we include, but I'd rather not sweat it right now.